phpdotnet/phd/Index.php

```diff
@@ -83,6 +83,10 @@ class Index extends Format
         'phpdoc'            => 'PI_PHPDOCHandler',
     );
 
+    private bool $inLdesc = false;
+    private bool $inSdesc = false;
+    private string $currentLdesc = "";
+    private string $currentSdesc = "";
```

Comment:
For what does the L and S stand for?

Reply:
Long and short. 

Long descriptions are text in `<title>` and `<refpurpose>` tags (e.g. class name descriptions (`'The DOMDocument class'`), function and method descriptions), short descriptions are text in `<refname>` and `<titleabbrev>` tags (e.g. class names (`DOMDocument`), function and method names).

Reply:
Please use those in the property name then, as I prefer a long and descriptive name than this cryptic name.

Reply:
I've followed the existing naming convention but in retrospect that wasn't a smart choice. :-)